ocs/defaults.py

```diff
@@ -66,6 +66,9 @@
     'bin_dir': './bin',
 }
 
+ENV_STATUS_PRE = {}
+ENV_STATUS_POST = {}
```

Comment:
do you suggest to move it to the exvironment_checker.py file?

Reply:
Yes, this would be better as it would prevent the state to be read or changed anywhere in the code. And it would make the code in env. checker self contained.

Reply:
done

Reply:
I don't see any change.

Reply:
Pushed now